man/docker-run.1.md

```diff
@@ -52,6 +52,7 @@ docker-run - Run a command in a new container
 [**--restart**[=*RESTART*]]
 [**--rm**[=*false*]]
 [**--security-opt**[=*[]*]]
+[**-s**|**--stop-signal**[=*SIGNAL*]]
```

Comment:
Here it's `--stop-signal`, but everywhere else it's `--kill-signal`.

Also, personally I prefer `--stop-signal` over `--kill-signal`. Since docker sends a `SIGKILL` after the 10 second timeout, calling this "kill-signal" makes it confusing which signal this is controlling.